implementations/rust/ockam/ockam_command/src/kafka/outlet/create.rs

```diff
@@ -27,26 +27,35 @@ use crate::{
 /// Create a new Kafka Outlet
 #[derive(Clone, Debug, Args)]
 pub struct CreateCommand {
+    /// Address of your Kafka Outlet, which is part of a route used in other commands.
+    /// This unique address identifies the Kafka Outlet worker on the Node on your local machine.
+    /// Examples are `/service/my-outlet` or `my-outlet`.
```

Comment:
Can those 2 be really used interchangeably? If `${name}` is going to be injected somewhere, it will have to be of one form or the other I guess? Unless we always prepend `/service` when it doesn't exist? Maybe we should have a dedicated type instead of `String` for those things.